flask_app/app.py

```diff
@@ -8,7 +10,32 @@
 
 CORS(app, supports_credentials=True, origins=['http://localhost:3000'])
 
-# Comment
+# @app.route('/api/test', methods=['GET']) 
+
+@app.route('/test')
+def test():
+    return jsonify({'message': 'Hello World!'})
+
+@app.route('/api')
+def run_factcheck_go():
+    try:
+        run = 'news'
+        command = ['-c', run] 
+        # sub_command = ['-q', 'JD Vance Couch']
+        sub_command = []
+
+        result = subprocess.run(['./go/go.exe'] + command + (sub_command if sub_command is not None else ''), capture_output=True, text=True, encoding='utf-8')
```

Comment:
@Stanley-Wang910  
this is really nice example / pattern IMO 
i don't think that will work in a unixy system though, right? 
we can look into that

Reply:
No probably not, I was just using it as an example to see if I could run Go Executables in Python. I'll do some polishing